crates/rspack_core/src/runtime_globals.rs

```diff
@@ -238,7 +236,6 @@ impl RuntimeGlobals {
   pub fn name(&self) -> &'static str {
     use RuntimeGlobals as R;
     match *self {
-      R::INTEROP_REQUIRE => "ir",
       R::EXPORT_STAR => "es",
```

Comment:
why remove this

Reply:
It is swc herlps, we don't need this at now